components/edit-collective/sections/PaymentReceipts.js

```diff
@@ -0,0 +1,270 @@
+    my={3}
+    key={index.toString()}
+    alignItems="center"
+    display="flex"
+    flexDirection={['column', 'row']}
+    justifyContent="space-between"
+    py={3}
+    px="24px"
+  >
+    <Flex alignItems="center">
+      <Avatar collective={invoice.host} borderRadius="16px" mr={3} size="48px" />
+      <Box>
+        <HostName
+          fontSize={['13px', '17px']}
+          lineHeight={['20px', '24px']}
+          letterSpacing={[null, '-0.16px']}
+          color="black.900"
+          fontWeight="500"
+          my={0}
+        >
+          <FormattedMessage id="Fiscalhost" defaultMessage="Fiscal Host" />:{invoice.host.name}
+        </HostName>
+        <Span
+          fontSize={['10px', '15px']}
+          lineHeight={['14px', '21px']}
+          letterSpacing={[null, '-0.1px']}
+          color="black.600"
+          fontWeight="400"
+          mt={0}
+        >
+          {`${invoice.month}/${invoice.year}`} - {invoice.totalTransactions}{' '}
+          <FormattedMessage
+            id="paymentReceipt.transaction"
+            values={{
+              n: invoice.totalTransactions,
+            }}
+            defaultMessage="{n, plural, one {Transaction} other {Transactions}}"
+          />
+        </Span>
+      </Box>
+    </Flex>
+    <StyledButton
+      lineHeight="16px"
+      fontSize="13px"
+      width="142px"
+      padding="4px 16px"
+      disabled={invoice.loadingInvoice}
+      mt={3}
+      borderColor="#C4C7CC"
+      onClick={() => {
+        invoice.downloadInvoice({
+          fromCollectiveSlug: invoice.fromCollective.slug,
+          toCollectiveSlug: invoice.host.slug,
+          dateFrom: invoice.dateFrom,
+          dateTo: invoice.dateTo,
+        });
+      }}
+    >
+      <FormattedMessage id="DownloadReceipt" defaultMessage="Download receipt" />
+    </StyledButton>
+  </StyledCard>
+);
+
+const Receipts = ({ invoices }) => {
+  const { loading: loadingInvoice, call: downloadInvoice } = useAsyncCall(saveInvoice);
+  const byMonthYear = groupBy(invoices, invoice => `${invoice.month}-${invoice.year}`);
+
+  return Object.keys(byMonthYear).map(monthYear => {
+    const dateMonth = dayjs.utc(`${byMonthYear[monthYear][0].year}-${byMonthYear[monthYear][0].month}`, 'YYYY-M');
+    const dateFrom = dateMonth.toISOString();
+    const dateTo = dateMonth.endOf('month').toISOString();
+
+    return (
+      <Flex key={monthYear} flexDirection="column">
+        <Flex alignItems="center" justifyContent="space-between">
+          <H3 fontSize="16px" lineHeight="24px" color="black.900">{`${dateMonth.format('MMMM')} ${dateMonth.format(
+            'YYYY',
+          )}`}</H3>
+          <Divider width={['60%', '80%']} borderBottom="1px solid #C4C7CC" />
+        </Flex>
+        {byMonthYear[monthYear]
+          .map(invoice => ({
+            ...invoice,
+            loadingInvoice,
+            downloadInvoice,
+            dateFrom,
+            dateTo,
+          }))
+          .map(renderReceiptCard)}
```

Comment:
:ice_cream: You could have a single map if you refactored `renderReceiptCard` as `ReceiptCard` component.
```suggestion
        {byMonthYear[monthYear]
          .map((invoice, index) => <ReceiptCard key={index} ...{{...invoice, loadingInvoice,downloadInvoice,dateFrom,dateTo}} />
          })
        }
```